Lib/ToolsDir/jasmine-boot330.ts

```diff
@@ -14,8 +14,7 @@ declare var jasmineRequire: any;
         oneFailurePerSpec: true,
         hideDisabled: false,
         specFilter: function(spec: any) {
-            //console.log("Filter "+spec.getFullName());
-            return true;
+            return new RegExp((<any>window.parent).specFilter).exec(spec.getFullName());
```

Comment:
Use `test` method instead of `exec`. Also create RegExp ahead so it does not need to be created again for each test.

Reply:
Changed.